api/services/elastigroup/azure/spotVms/paths/importFromVirtualMachine.yaml

```diff
@@ -6,12 +6,13 @@ get:
   tags:
     - Elastigroup Azure Spot VMs
   parameters:
-    - in: path
+    - $ref: "../../../../../commons/parameters/accountId.yaml"
+    - in: query
```

Comment:
wrong change.
https://api.spotinst.io/azure/compute/group/import/resourceGroup/{**resourceGroupName**}/virtualMachine/{**virtualMachineName**}

same for line 15